Sync and app menu updates for M100 (#1757)
MicrosoftDocs/edge-developer: microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

As we test and gradually release this feature over the coming weeks, users who have opted in to sync will start to see a **Your available apps** section in the `edge://apps` page, and a redesigned **Apps** menu.

### The _Your available apps_ section

The **Your available apps** section will show any apps you've installed on other devices with the same signed-in profile, that aren't currently installed in the Microsoft Edge instance you're using.  The available apps will be shown in their own section with gray icons.  Click the **Install** button on an app to install the app on that device:

![Available apps shown on Apps page with install button highlighted.](media/available-app-install.png)

App sync is available on desktop devices only, at this stage.  To keep your apps in sync across devices, adding an app on one device will add it to the **Your available apps** section on other devices.  Uninstalling an app on one device will remove it from other devices.

### Redesigned Apps menu

> In Microsoft Edge, the **Apps** command in the **Settings and more** menu is being redesigned:

Reviewer: There is inconsistency with "Apps" vs "App" menu (I didn't notice until it was bolded). Should they all be "Apps" since that's used in the toolbar button's tooltip and as the title of the hub?

Reviewer 2: where the UI says "Apps", the article should say **Apps**.  re: Apps UI items

> *  The **App** menu icon can be added to the toolbar, for quick access.

Reviewer 2: ```suggestion
*  The **Apps** menu icon can be added to the toolbar, for quick access.
```

*  The popup menu can be pinned open as a sidebar to show your apps alongside web content.

> *  Customizable view and sort options have been added, along with a **Your available apps** section to support the App Sync feature.

Reviewer: The current UI says "My available apps" although I think the new figmas may just have "Available apps" as the title. The edge://apps page has slightly different wording than the hub.

Author: Amanda is correct - in the hub menu the section is My apps and My available apps.

> *  Customizable view and sort options have been added, along with a **Your available apps** section to support the App Sync feature.

Reviewer: ```suggestion
*  Customizable view and sort options have been added, along with a **My available apps** section to support the App Sync feature.
```

> *  Customizable view and sort options have been added, along with a **Your available apps** section to support the App Sync feature.

Reviewer 2: ```suggestion
*  Customizable view and sort options have been added, along with a **My available apps** section to support the App Sync feature.
```

> Users can show the **App** menu icon on the toolbar, for quick access:

Reviewer 2: ```suggestion
Users can show the **Apps** menu icon on the toolbar, for quick access:
```

> [!div class="mx-imgBorder"]

> > ![App icon can be shown on the toolbar, for quick access.](media/apps-toolbar-icon.png)

Reviewer 2: ```suggestion
> ![The Apps icon can be shown on the toolbar, for quick access.](media/apps-toolbar-icon.png)
```

Users can pin the **Apps** menu open as a sidebar:

> [!div class="mx-imgBorder"]

> > ![Users can pin the apps menu open as a sidebar.](media/pin-apps-menu.png)

Reviewer 2: ```suggestion
> ![Users can pin the Apps menu open as a sidebar.](media/pin-apps-menu.png)
```

<!-- ====================================================================== -->
## What's New in Microsoft Edge 100

> Microsoft Edge Canary/Dev channels moved to version 100 on February 9,2022 and in this milestone, we are testing the following new web app features. 

Reviewer 3: Should we expand a bit on what "we are testing" means? What should readers expect to see in 100? Will they get the new features, or not? It might help to set expectations here.

### App Sync

> The sync service powering features like favorites, settings, history & more has been expanded to include your installed web apps - that is PWAs and sites installed as apps. The sync service works when you are signed in to your profile and have opted in to syncing. As we test and gradually release this feature over the coming weeks, users that have opted in to sync will start to see "Available apps" in the `edge://apps` page and the [new Apps menu](#redesigned-apps-menu). 

Reviewer 3: I would suggest linking to https://support.microsoft.com/en-us/microsoft-edge/sign-in-to-sync-microsoft-edge-across-devices-e6ffa79b-ed52-aa32-47e2-5d5597fe4674 to make clearer to readers about how to opt-in to sync.

Reviewer 3: ```suggestion
The sync service powering features like favorites, settings, history & more has been expanded to include your installed web apps - that is PWAs and sites installed as apps. The sync service works when you are signed in to your profile and have opted in to syncing. As we test and gradually release this feature over the coming weeks, users that have opted in to sync will start to see the **Your available apps** section in the `edge://apps` page and the [new Apps menu](#redesigned-apps-menu). 
```

> Available apps will show any apps you've installed on other devices with the same signed in profile, that aren't currently installed in the Edge instance you are using. The available apps will be shown in their own section with grey icons. Press the "Install" button to install the app on that device.

Reviewer 3: ```suggestion
The **Your available apps** section will show any apps you've installed on other devices with the same signed in profile, that aren't currently installed in the Edge instance you are using. The available apps will be shown in their own section with grey icons. Click **Install** to install the app on that device.
```

>    :::image type="content" source="media/available-app-install.png" alt-text="Available apps shown on Apps page with install button highlighted.":::

Reviewer 3: Note: there is no need to indent these images. Also, @mikehoffms can confirm, but we've been moving towards using normal markdown image tagging (i.e. `![alt text](/path/to/img/png)`) instead of the `:::image:::` extension.

Reviewer 2: I'm fixing as part of routine Writer/Editor pass

> To keep your apps in sync across devices, adding an app on one device will add it to the "Available apps" on other devices. Uninstalling an app on one device will remove it from other devices.

Reviewer: nit/personal preference: What do you think about changing to "adding an app on one device will add it to the "Available apps" [section] on other devices." ?

> App sync is available on desktop devices only at this stage. To keep your apps in sync across devices, adding an app on one device will add it to the "Available apps" section on other devices. Uninstalling an app on one device will remove it from other devices.

Reviewer 3: ```suggestion
App sync is available on desktop devices only at this stage. To keep your apps in sync across devices, adding an app on one device will add it to the **Your available apps** section on other devices. Uninstalling an app on one device will remove it from other devices.
```

### Redesigned Apps Menu

> The Apps menu item within Edge's "More Menu" is being redesigned so that the menu icon can be added to the taskbar for quick access and the menu can be pinned open as a sidebar to show your apps alongside web content. Customizable sort and view options have been added along with an "Available apps" section to support the app sync feature described above.

Reviewer: 1. The tooltip for the "..." menu is "Settings and more". Should this say "Edge's 'Settings and More menu'" instead?
2. Should "taskbar" be "toolbar" instead? In code, that area is referred to as a toolbar, but I'm not sure what it's referred to in docs. If it is a "taskbar", should we call it a "browser taskbar" instead to differentiate it from the system taskbar?
3. The hub can be pinned to the sidebar, but it also exists as a popup by default. Do we typically refer to hubs as "sidebars" or "hubs"?

Author: thanks - changed to toolbar, Settings and more, added popup.

> The Apps menu item within Edge's "Settings and more" menu is being redesigned so that the menu icon can be added to the toolbar for quick access and the popup menu can be pinned open as a sidebar to show your apps alongside web content. Customizable sort and view options have been added along with an "Available apps" section to support the app sync feature described above.

Reviewer 3: ```suggestion
The **Apps** menu item within Microsoft Edge's **Settings and more** menu is being redesigned so that the menu icon can be added to the toolbar for quick access and the popup menu can be pinned open as a sidebar to show your apps alongside web content. Customizable sort and view options have been added along with an **Your available apps** section to support the app sync feature described above.
```

   :::image type="content" source="media/apps-toolbar-icon.png" alt-text="App icon can be shown on the toolbar for quick access":::

> Users can choose to show the app icon on the toolbar for quick access.

Reviewer: The paragraph above says "taskbar". If you choose to keep that terminology, then this one should match.

>    :::image type="content" source="media/pin-apps-menu.png" alt-text="Users can pin the apps menu open to the side of Edge":::

Reviewer: nit: The alt text reads weird to me, especially since "Edge" can also mean the edge of something. Note that screen reader users don't get the benefit of knowing that Edge is capitalized. What do you think about something like "Users an pin the apps menu/hub open as a sidebar"?

> Users can pin the apps menu open

Reviewer: Nit: should end with a period.

Author: added period

> Users can pin the apps menu open

Reviewer: Nit: The hub is not really a menu, so it doesn't feel correct to be referring to it as an "apps menu" here. Can this be "apps hub" instead?

Author: I like as a sidebar so I'll change it to say that. We know this UI as a hub but I don't think it's well known as a descriptor outside of our team. Sidebar is more informative IMHO.
